tests/dummy/app/router.js

```diff
@@ -8,5 +8,4 @@ export default class Router extends EmberRouter {
 
 Router.map(function () {
   this.route('four-oh-four');
```

Comment:
```suggestion
  // no routes
```

Reply:
We actually use this route for testing. We might be able to clean it up eventually.